apps/dashboard/app/views/module_browser/_module_table.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="table-responsive">
+  <table class="table">
```

Comment:
I'm quite sure we don't want lists within a table, for accessibility reasons. You'd end up with 100 different lists which may be confusing to screen readers.

What else could we do you ask? I'm not entirely sure. I'm thinking maybe just one giant list?  I'm not asking you to make changes now, but I think we do need a better HTML structure.